vitrina/structure/views.py

```diff
@@ -1456,6 +1457,7 @@ def form_valid(self, form):
         visibility = form.cleaned_data.get("visibility")
         status = form.cleaned_data.get("status") or Status.objects.filter(is_default=True).first()
         eli = form.cleaned_data.get("eli")
+        draft_version, created = _Version.objects.get_or_create(dataset=self.dataset, status=VersionStatus.DRAFT)
```

Comment:
```suggestion
        draft_version, _ = _Version.objects.get_or_create(dataset=self.dataset, status=VersionStatus.DRAFT)
```

if you not going to use it.

Reply:
Agree with the suggestion, but an import already takes _ as a name
`from django.utils.translation import gettext_lazy as _`

Reply:
@Gustascct just for interest - does this cause name collisions in your case?

Reply:
The created bool returned by get_or_create overwrites the _ and then all the _("String") fail because of that.